engine/language-server/src/main/scala/org/enso/languageserver/runtime/ContextRegistryProtocol.scala

```diff
@@ -505,6 +505,13 @@ object ContextRegistryProtocol {
     */
   case class ModuleNotFound(moduleName: String) extends Failure
 
+  /** Signals that the module containing the provided expression id cannot be
+    * found.
+    *
+    * @param expressionId the expression id
+    */
+  case class ModuleNotFoundForExpression(expressionId: UUID) extends Failure
```

Comment:
Wouldn't it be easier to merge this message with `ModuleNotFound` and make `expressionId: Option[UUID]`?

Reply:
We still need a separate JSONRPC message, so I prefer 1:1 correspondence

Reply:
> We still need a separate JSONRPC message

Do we? Can't we reuse the existing one?

Probable answer: no, we cannot as we don't have _module name_ - if there was a shared message, it would have to have both attributes - e.g. _module name_ as well as _expressionId_ - optional.